freestyle/shared/src/main/scala/freestyle/package.scala

```diff
@@ -111,8 +105,8 @@ package object freestyle {
      */
     def freeS: FreeS[F, A] = FreeS.liftPar(fa)
 
-    def exec[G[_]: Applicative](implicit interpreter: F ~> G): G[A] =
```

Comment:
This change was used to avoid a strange error in the test for implicit resolution messages.